proto/storage/compliance_operator_v2.proto

```diff
@@ -135,7 +135,6 @@ message ComplianceOperatorCheckResultV2 {
     string                    standard         = 12 [(gogoproto.moretags) = 'search:"Compliance Standard,hidden,store"'];
     string                    control          = 13;
     string                    scan_id          = 14 [(gogoproto.moretags) = 'search:"-" sql:"fk(ComplianceOperatorScanV2:id),no-fk-constraint"'];
-    string                    scan_config_id   = 16 [(gogoproto.moretags) = 'search:"Compliance Scan Config ID,hidden,store" sql:"fk(ComplianceOperatorScanConfigurationV2:id),no-fk-constraint,type(uuid)"'];
     string                    scan_config_name = 17;
```

Comment:
```suggestion
    string                    scan_config_name = 16;
```

Reply:
also adjust the comment at the top of the object with the next tag.  That should become 17